docs/src/layouts/MainLayout.vue

```diff
@@ -243,27 +108,25 @@ $hide-social-links-viewport: 862px;
   }
 }
 
-// remove some children just before xs
-.toolbar-menu-items {
-  .q-btn:nth-last-child(-n+3) {
-    @media screen and (max-width: $adjust-header-viewport) {
-      display: none;
-    }
-  }
+.doc-left-drawer {
+  overflow: inherit !important;
```

Comment:
This feels strange, why is it needed?

Reply:
I couldn't also figure out exactly why this style had been used. I'm using the exact same drawer and app-menu that was used in the Doc layout, and for some reason that style had been added in the doc layout. I'm thinking it should have been added on purpose, but just could not reproduce the error that is supposed to help him.

Reply:
If you cannot reproduce it, remove it.
It will either come around again, so you can document the why, or it's not needed anymore for some reason.
Checking out the commit which added it also helps in those cases